src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs

```diff
@@ -44,11 +44,11 @@ await using (var stream = dataFile.Open())
                     releaseId: releaseId,
                     stream: stream,
                     fileType: ReleaseFileTypes.Data,
-                    fileName: dataFile.Name,
+                    fileName: dataFile.Name.ToLower(),
                     contentType: "text/csv",
                     metaValues: GetDataFileMetaValues(
                         name: blob.Name,
```

Comment:
Thinking about it, shouldn't this be the subject name?

Reply:
No that's OK - The subject is set on this line...
name: blob.Name